lib/rules/web/admin_console/4.3/goto.xyaml

```diff
@@ -99,6 +99,10 @@ goto_catalog_page:
 goto_create_app_from_imagestream_page:
   url: catalog/source-to-image?imagestream=<is_name>&imagestream-ns=openshift&preselected-ns=<project_name>
 
+# monitoring
+goto_monitoring_metrics_page:
+  url: monitoring/query-browser?query0=
+
```

Comment:
How about `monitoring/query-browser` for the url?

Reply:
visiting `monitoring/query-browser` will redirect to `monitoring/query-browser?query0=`. I think I can change it to a simpler one

Reply:
updated